cmd/mdatagen/loader.go

```diff
@@ -194,7 +194,7 @@ func (a attribute) TestValue() string {
 	case pcommon.ValueTypeSlice:
 		return fmt.Sprintf(`[]any{"%s-item1", "%s-item2"}`, a.FullName, a.FullName)
 	case pcommon.ValueTypeBytes:
-		return fmt.Sprintf(`bytes("%s-val")`, a.FullName)
+		return fmt.Sprintf(`[]byte("%s-val")`, a.FullName)
```

Comment:
Is it possible to unit test these? Seems like a really basic thing which wasn't caught by the extensive CI.

Reply:
There aren't tests for this at this point. It's not a bad idea, but given the facts that this is only used for generating test code, it shouldn't change very frequently, and it can't build when something is wrong, I don't see as much value for this specific test case at this point. I'm open to feedback from others though.